common/src/env/cpp/Env.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+#include "Env.h"
+
+#include <cstdlib>
+
+#ifdef __APPLE__
+
+void setEnv(const char* name, const char* value) {
+  setenv(name, value, 1);
+}
```

Comment:
Is this different from default implementation (after `#else`)?

Reply:
Copy/paste artefact - I'll merge macos/linux impls.